Dockerfile

```diff
@@ -1,14 +1,9 @@
-FROM azul/zulu-openjdk-alpine:8
-MAINTAINER Julien Ponge <julien.ponge@insa-lyon.fr>
-
-RUN apk add --no-cache
-
+FROM azul/zulu-openjdk:8 AS builder
 ADD . /src
+WORKDIR /src
+RUN ./gradlew installDist
 
-RUN cd /src &&\
-  ./gradlew -v &&\
-  ./gradlew installDist &&\
-  mkdir -p /opt/golo &&\
-  cp -R /src/build/install/golo/* /opt/golo &&\
-  ln -s /opt/golo/bin/golo /usr/bin/golo &&\
-  rm -rf /src /root/.gradle
+FROM azul/zulu-openjdk-alpine:8
+COPY --from=builder /src/build/install/golo /opt/golo
+RUN ln -s /opt/golo/bin/golo /usr/bin/golo
```

Comment:
Why don't you add to PATHs instead of symlink ?
- `/opt/golo/man` into `MANPATH`
- `/opt/golo/bin` and maybe `/opt/golo/share/shell-completion/` into `PATH`
- maybe `opt/golo/lib/*.jar` to `CLASSPATH`

Reply:
Good idea